honeybee_3dm/helper.py

```diff
@@ -1,4 +1,41 @@
 """A collection of helper functions."""
+import json
+from enum import Enum, unique
+
+@unique
+class HB_layers(Enum):
+    """Default Honeybee layers.
+    
+    This class hosts all the default Honeybee layers in
+    layer key : layer name structure
+
+    Args:
+        Enum: An Enum object.
+    """
+    grid = 'HB_grid'
+    room = 'HB_room'
+    wall = 'HB_wall'
+    roof = 'HB_roof'
+    floor = 'HB_floor'
+    airwall = 'HB_airwall'
+    shade = 'HB_shade'
+    aperture = 'HB_aperture'
+    door = 'HB_door'
+    view = 'HB_view'
+
+
+def read_json(path):
+    """Get a dictionary from a config.json file.
+
+    Args:
+        path: A text string for the path to the config.json.
+
+    Returns:
+        A dictionary for the config.json
+    """
+    with open(path) as fh:
+        config = json.load(fh)
+        return config
```

Comment:
It's ok to keep this but honestly I think just writing the code for loading the JSON data is just as good.